hashicorp/boundary-ui
# feat: Prompt before quitting inprogress session on desktop client quit

<img width="372" alt="prompt" src="https://user-images.githubusercontent.com/111036/118012064-abf0ae00-b31e-11eb-91d7-f36fda465d96.png">

To test:

1. Launch desktop client and connect to a boundary controller
2. Establish multiple sessions to targets
3. Verify creation of multiple boundary process using 'Activity Monitor' or 'ps'.
4. Quitting desktop client will show a new prompt

When 'Close & Quit' option is chosen, verify that all previously created processes no longer exist after desktop quits.
When 'Ignore & Quit' option is chosen, verify that all previously created processes still exist after desktop quits.
When 'Quit' option is chosen, verify that desktop will not quit..
